www/base/src/components/MultipleWorkersActionsModal/MultipleWorkersActionsModal.tsx

```diff
@@ -65,6 +66,18 @@ export const MultipleWorkersActionsModal = observer(({workers, preselectedWorker
     }
   }
 
+  const onInputChange = (
+    inputValue: string,
+    { action, prevInputValue }: InputActionMeta
+  ) => {
+    if (action === 'input-change') return inputValue;
+    if (action === 'menu-close') {
+      if (prevInputValue) setMenuIsOpen(true);
```

Comment:
Please add explicit braces. Maybe `setMenuIsOpen(prevInputValue ? true : undefined)` could also be concise yet readable enough?